ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -290,6 +308,26 @@ tests:
     name: osde2e-credentials
   - mount_path: /usr/local/osde2e-rosa-stage
     name: osde2e-rosa-stage
+- as: hypershift-stage-e2e-default
+  commands: |
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="hypershift"
+    export CHANNEL="candidate"
+    export CLUSTER_VERSION="4.12.0-rc.8"
```

Comment:
Can we set this to latest 4.12? This will fall out of date quickly and I don't see much value in continuously testing the same build for weeks at a time

Reply:
Hadn't seen this. 
Updated to 4.12.0, I would leave this to be updated to the latest but it is gated by Rosa: 
ERR: Expected a valid OpenShift version: A valid version number must be specified
Valid versions: 4.12.0 4.12.0-rc.8 4.12.0-rc.7 4.12.0-rc.6 4.12.0-rc.5 4.12.0-rc.4 4.12.0-rc.3 4.12.0-rc.2 4.12.0-rc.1 4.12.0-rc.0

I'm not sure how they approve hypershift versions so we can align with them but the static version is problematic as versions move up.

Reply:
Hmm, I think `4.12.0` is possibly older than `4.12.0-rc.8`. Is it possible that we set this to install latest 4.12 at any given time?

Reply:
Tested on my end the stable channel is able to pick up the latest 4.12.0 and it is golden. 
When picking versions for the candidate channel though it ask for specific rc-x.